canon/src/lib.rs

```diff
@@ -7,23 +7,20 @@
 //! Canonical, a no_std, host-allocating serialization library
 #![cfg_attr(not(feature = "host"), no_std)]
 #![feature(never_type)]
+#![feature(thread_local)]
```

Comment:
How's stable / safe to use this feature across platforms? (e.g. Linux, MacOS, Windows, wasm32)